pkg/kubemark/hollow_kubelet.go

```diff
@@ -18,6 +18,7 @@ package kubemark
 
 import (
 	"fmt"
+	"k8s.io/utils/mount"
```

Comment:
This new dep doesn't appear to be used anywhere?

Reply:
+1, I believe lint will catch this